pandas/core/indexing.py

```diff
@@ -222,7 +223,7 @@ def _validate_key(self, key, axis: int):
         """
         raise AbstractMethodError(self)
 
-    def _has_valid_tuple(self, key):
+    def _has_valid_tuple(self, key: tuple):
```

Comment:
Can you use the generic `Tuple` from the typing module? Providing the type(s) contained within would be even better (maybe `Tuple[Hashable]`?)